src/components/Trigger/Trigger.tsx

```diff
@@ -54,8 +52,8 @@ export const Trigger: FC<TriggerProps> = ({
             {children}
             <div
                 className={merge([
-                    "tw-flex-none tw-flex tw-items-center",
-                    size === TriggerSize.Large ? "tw-pr-5 tw-gap-1.5" : "tw-pr-3 tw-gap-1",
+                    "tw-flex-none tw-flex tw-items-center tw-absolute",
```

Comment:
This will also affect the spacing on the multi-select and the color picker. Although from tests the color picker padding already does the job of providing enough space to the caret on the right side, the multi-select should also be updated to accommodate this change

Reply:
Just tried both the Color Picker and the Multi Select components and they both seem to have the correct spacing on the right of the caret icon.
Could you verify as well?

Reply:
Multi Select has the correct spacing on the right of the icon, however there is a small chance that the multi select options could cover the caret icon if the input is full of options

Reply:
Ah gotcha!
Let me see if something can be done about this.

Reply:
Please take a look at this:

https://user-images.githubusercontent.com/26199969/151532476-25d2853f-dec6-41d1-9bd4-3c9874df2c52.mov

It seems that it doesn't affect the Multi Select component.
There might be some other edge cases that I'm missing though.

Reply:
<img width="491" alt="Screenshot 2022-01-28 at 11 52 01" src="https://user-images.githubusercontent.com/93908356/151534590-fe22fb34-c4d9-4a74-a3ef-6c5d84b33a2e.png">
Hopefully this demonstrates the issue :)

Reply:
It's only a small visual glitch, but might as well fix it in this pr

Reply:
Thanks, makes sense now.

Reply:
Should be fixed now.

Reply:
looks great 😄